packages/openapi-to-markdown/src/components/XmlOrJson.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { OpenAPIV3_1 } from '@scalar/openapi-types'
+// @ts-expect-error Doesn’t come with types
```

Comment:
🫨 in this day and age?

Reply:
yeah, I should look for an alternative. but it’s used in another package already, so that was easy.